### PR TITLE
kernel: Add support for expand after current line

### PR DESCRIPTION
**Pull request by garazdawi**
This PR changes the tab-completion location to be after the current cursor instead of before. It is possible to configure the behaviour to work as it did before by setting the `stdlib` application parameter `shell_expand_location` to either `above` or `below`.

[asciicast](https://asciinema.org/a/4tDMbPksWndZ6vApbsOUl1viL)

### CODE REVIEW

**Review by frazze-jobb — COMMENTED** [review]
I tested it, it works. The only problem is that when the expansion results are to big it doesnt move the cursor, instead the cursor is in the middle of the expansion result.
I think what we want to do is just write the maximum amount of lines available.
Press tab again, and you enter inside the expansion buffer area and you may use arrow keys pageup, page down etc. But that can be another commit.

File: lib/ssh/src/ssh_cli.erl [hunk 1]
```diff
@@ -670,7 +670,9 @@ start_shell(ConnectionHandler, State) ->
             {_,_,_} = Shell ->
                 Shell
         end,
-    State#state{group = group:start(self(), ShellSpawner, [{echo, get_echo(State#state.pty)}]),
+    State#state{group = group:start(self(), ShellSpawner,
+                                    [{expand_below, false},
```
[the remark this answers is not in the document]
Comment (garazdawi) [reply lib/ssh/src/ssh_cli.erl:674]: We probably want to have ssh working, but I left it out for now.

**Comment by garazdawi** [comment]
> I think what we want to do is just write the maximum amount of lines available.
> Press tab again, and you enter inside the expansion buffer area and you may use arrow keys pageup, page down etc. But that can be another commit.

I added a commit that limits the number of rows printed to 5 and then if the user tabs again it shows all results. If all results are more lines than fit on the screen the expansions are shown above instead of below.

**Review by frazze-jobb — COMMENTED** [review]
Works great, good compromise to output to long completions above the cursor

File: lib/kernel/src/prim_tty.erl [hunk 2]
```diff
@@ -319,6 +322,16 @@ init(State, {unix,_}) ->
                    false -> (#state{})#state.position
                end,
 
+    %% According to the manual this should only be issues when the cursor
```
Comment (garazdawi) [inline lib/kernel/src/prim_tty.erl:325]:
```suggestion
    %% According to the manual this should only be issued when the cursor
```

**Pull request merged**